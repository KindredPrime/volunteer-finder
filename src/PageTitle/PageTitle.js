import React from 'react';

function PageTitle(props) {
  return (
    <div className="PageTitle">
      <h1>{props.title}</h1>
    </div>
  );
}

export default PageTitle;