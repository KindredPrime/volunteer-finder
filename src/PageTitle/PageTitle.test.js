import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import PageTitle from './PageTitle';

describe('PageTitle Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<PageTitle />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<PageTitle title="title" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});