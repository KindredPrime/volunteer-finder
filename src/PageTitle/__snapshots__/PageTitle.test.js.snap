// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Component renders the UI as expected 1`] = `
<div
  className="PageTitle"
>
  <h1>
    title
  </h1>
</div>
`;
